Userland/Libraries/LibGfx/Bitmap.h

```diff
@@ -88,13 +92,12 @@ enum RotationDirection {
 };
 
 class Bitmap : public RefCounted<Bitmap> {
+    friend class Media::ImageDecoder;
```

Comment:
Absolutely not. We cannot have cross-library friendships. That's bad layering. What's the need here? Why can't ImageDecoder use the public API?